### ISSUE TITLE
Weak contrast of "New [...]" buttons under attachments

### ISSUE THREAD

**Issue by raphj** [xwikisas/application-onlyoffice-connector#92]
**Describe the bug**

Firefox reports a too weak contrast for the "New Spreadsheet" button with the following class: `btn btn-success btn-oosheet oONewSpreadsheet`

I also find it relatively difficult to read the "New presentation" button (`btn btn-warning btn-ooprez oONewPresentation`) (even if its contrast is graded AA by Firefox).

The following should probably be done:
 - increase the contrast of these two buttons. Likely by dimming the background, as the meaning is entirely conveyed by the text anyway. The background of the first button ("New Document") might have to be adjusted to keep harmony / consistency
 - the use of `btn-warning`, `btn-primary` and `btn-success` classes to color the buttons is not appropriate here. These should be used to convey a warning, a primary or a success. If colors have to be kept, they should be done using another way.

**Screenshots**

<img width="461" height="153" alt="Image" src="https://github.com/user-attachments/assets/42786094-c957-42ef-94dc-96858e177953" />

**Environment (please complete the following information):**
 - OS: openSUSE Tumbleweed
 - Browser Firefox
 - XWiki version 17.10.2
 - App version 2.5.1

**Comment by sarahxb** [comment]
@raphj 
With the current change of adding ```color: #fff ``` (https://github.com/xwikisas/application-onlyoffice-connector/pull/97) , this is the result:

<img width="526" height="165" alt="Image" src="https://github.com/user-attachments/assets/f4c4a93f-96ac-4c61-b322-c4b01db8c8e6" />
I know this is not the mentioned fix idea, but I tried lightening the background and  it doesn't look too nice.
The only thing is the hover over the ```New Document``` button, which should be adjusted so that the text is more readable. 

Let me know if this is not an appropriate fix and I will try again with adjusting the background if so :) .

**Comment by raphj** [comment]
@sarahxb increasing the contrast by changing the color is good too. As long as the accessibility tools don't complain anymore... :-)
